src/js/view/error-container.ts

```diff
@@ -1,15 +1,28 @@
 import errorContainerTemplate from 'templates/error';
 import { createElement } from 'utils/dom';
 import { style } from 'utils/css';
+import { OS } from 'environment/environment';
+import TizenErrorContainer from 'view/controls/tizen/tizen-error-container';
 import type Model from 'controller/model';
 import type { PlayerError } from 'api/errors';
+import type { PlayerAPI } from 'types/generic.type';
 
-export default function ErrorContainer(model: Model, error: PlayerError): HTMLElement {
+require('css/error.less');
```

Comment:
why the additional css require here? Injecting CSS is not cheap (it triggers reflow and repaint). The less files are bundled together so that CSS can all be injected once, or twice as needed.

This should be included in either jwplayer.less or less that is imported into a tizen only loaded js module (require.ensure). I think if the tizen app is broken enough to encounter a setup error the standard error container might be fine, and you can deal with navigation in the app, rather than in the player.

Reply:
So the tizen-controls should require('tizen.less'). tizen.less should not be bundled in controls.less. Otherwise you're adding tizen styles to every player instance instead of only the ones running on tizen loading the tizen controls module.